infra/gcp/terraform/k8s-infra-oci-proxy-prod/oci-proxy.tf

```diff
@@ -62,12 +47,14 @@ resource "google_project_iam_member" "k8s_infra_oci_proxy_admins" {
   member  = "group:k8s-infra-oci-proxy-admins@kubernetes.io"
 }
 
+
 resource "google_service_account" "oci-proxy" {
   project      = google_project.project.project_id
-  account_id   = "oci-proxy-prod"
+  account_id   = "oci-proxy-sandbox"
```

Comment:
why change the name of the service account ?

Reply:
bad copy & paste, fixing now